ops/op2/signature.rs

```diff
@@ -658,6 +662,8 @@ pub enum SignatureError {
   InvalidWherePredicate(String),
   #[error("State may be either a single OpState parameter, one mutable #[state], or multiple immultiple #[state]s")]
   InvalidOpStateCombination,
+  #[error("JsRuntimeState may only be used in one parameter")]
+  InvalidMultipleJsRuntimeState,
```

Comment:
Love it. This should probably happen for the other single cardinality params too.